pendingchanges/nanovdb.txt

```diff
@@ -0,0 +1,33 @@
+NanoVDB:
+
+Highlights:
+- NanoVDB is no longer limited to static applications, such as rendering, but can increasingly be applied to problems that
+  involve dynamic topology, such as level set tracking, grid building, morphology operations and merging of grids. This is
+  due to the fact that we keep adding more and more tools to NanoVDB that facilitates tree modification. However, unlike
+  OpenVDB, these operations generate new NanoVDB grids (as oppose to in-palace modifications in OpenVDB), yet are typically
+  much faster since they are performed on the GPU. Many of these exciting new features are the foundation of fVDB.
+
+New features:
+- Added nanovdb/tools/cuda/DilateGrid.cuh, which performs topology dilation on the GPU.
+- Added nanovdb/tools/cuda/MergeGrids.cuh, which merges the topology of two grids on the GPU.
+- Added nanovdb/tools/cuda/CoarsenGrid.cuh, which performs topological coarsening of grids on the GPU.
+- Added nanovdb/tools/cuda/RefineGrid.cuh, which performs topological refinement of grids on the GPU.
+- Added nanovdb/tools/cuda/PruneGrid.cuh, which prunes the topology of a grid using a sidecar mask on the GPU.
+- Added nanovdb/util/cuda/Injection.cuh, which facilitates sidecar data copying - on the GPU - in the overlap of two grids with different topology.
+- Added Mat classes and other math types to NanoVDB.
+- Major rewrite of nanovdb::NanoToOpenVDB, which now allows nanovdb::IndexGrids and blind (or sidecar) data to be converted into an openvdb grid.
+
+Improvements:
+- Improvements to mGPU grid building and introduction of a separate test executable for mGPU tests
```

Comment:
(Optional). Consider mentioning:
- Added support for `LevelSet`, `FogVolume`, and `Staggered` grid semantic. Added `toGridClass` and `toSemantic` to map Grid Blind data semantic to Grid class and vice versa.